stdlib/public/runtime/Reflection.mm

```diff
@@ -1068,62 +1068,57 @@ void swift_ObjCMirror_subscript(String *outString,
   
 // -- MagicMirror implementation.
 
-// TODO: There are other non-Apple platforms that underscore asm symbols.
-#if defined(__APPLE__)
-# define UNDERSCORE "_"
-#else
-# define UNDERSCORE
-#endif
+#define GLUE_EXPANDED(a,b) a##b
+#define GLUE(a,b) GLUE_EXPANDED(a,b)
+#define SYMBOL_NAME(name) GLUE(__USER_LABEL_PREFIX__,name)
```

Comment:
@jckarter I don't think we would support building the runtime with MSVC ever.  `clang-cl`, sure, but not MSVC.